content/docs/command-reference/push.md

```diff
@@ -185,7 +185,7 @@ Finally, we used `dvc status` to double check that all data had been uploaded.
 ## Example: What happens in the cache?
 
 Let's take a detailed look at what happens to the
-[cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory)
+[cache directory](/doc/user-guide/concepts/dvc-cache#structure-of-the-cache-directory)
```

Comment:
Kind of unrelated to the PR but we probably don't need this link since it's also in a block quote by the end of the example. But maybe could be a tooltip? If there aren't other tooltips to this concept nearby already.

Reply:
Agreed. Added the tooltip here and added `cache directory` to the available matches in the concept.

There is another `cache` tooltip in the next paragraph. Too many?

Reply:
I'd just use `<abbr>cache</abbr> directory` and avoid the extra glossary match entry.

> There is another cache tooltip in the next paragraph. Too many?

Yep, I'd remove that later one. Ugh managing `<abbr>` tags can be painful.